src/main/scala/pl/project13/scala/akka/raft/Candidate.scala

```diff
@@ -64,7 +91,7 @@ private[raft] trait Candidate {
       if (leaderIsAhead) {
         log.info("Reverting to Follower, because got AppendEntries from Leader in {}, but am in {}", append.term, m.currentTerm)
         m.clusterSelf forward append
-        goto(Follower) using m.forFollower
+        goto(Follower) using m.forFollower()
```

Comment:
FWIW, with a default parameter (as @dmitraver added), it doesn't compile without parens